plugins/webhooks/index.js

```diff
@@ -253,7 +253,10 @@ async function triggeredPipelines(pipelineFactory, scmConfig, branch, type, acti
     let pipelinesOnOtherBranch = [];
 
     pipelines.forEach((p) => {
-        if (p.scmUri.startsWith(scmUri)) {
+        const tmpScmUri = p.scmUri.split(':');
```

Comment:
I think we should add a comment why splitting and concatenation of `scmUri`.

Reply:
Thanks, I added comment.